app/scripts/react/components/Checkout/CheckoutPayments.jsx

```diff
@@ -20,6 +20,9 @@ class CheckoutPayments extends Component {
             />
             <div className="b-cart__form__payment-name">
               {item.title}
+              {item.show_icon &&
+                <img src={item.icon_url} />
```

Comment:
тогда теме более хорошо бы указать в тебе высоту (height)

On Thu, Jul 14, 2016 at 9:35 PM Alexander <notifications@github.com> wrote:

> In app/scripts/react/components/Checkout/CheckoutPayments.jsx
> <https://github.com/BrandyMint/kiosk-public-static-v2/pull/222#discussion_r70859429>
> :
>
> > @@ -20,6 +20,9 @@ class CheckoutPayments extends Component {
> >              />
> >              <div className="b-cart__form__payment-name">
> >                {item.title}
> > +              {item.show_icon &&
> > +                <img src={item.icon_url} />
>
> размер изображения? их же не пользователи загружают. я их все порезал
> одинаковой вышины
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BrandyMint/kiosk-public-static-v2/pull/222/files/027f2d2f8bedd8a62c9ad37b636f992413928e0d#r70859429>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACqOTh0ZWg18HS1oA3KnORzSnyXl79lhks5qVoFtgaJpZM4JMLBs>
> .
>